45-add-vms.sh

```diff
@@ -26,6 +26,19 @@ for i in $(seq 1 1 $WORKERS); do
     add_dns worker-$i
 done
 
+if [[ $# -gt 1 && "numa" == $(echo "$1" | tr '[:upper:]' '[:lower:]') && $2 -gt 0 ]]; then
```

Comment:
since there are not other parameters i would just use 
NUMA_NODES=$1
[[ "$NUMA_NODES" ]] && NUMA_MEMORY=$(( $WORKER_MEMORY / $NUMA_NODES ))
for j in $(seq 1 $NUMA_NODES); do ....